### PR TITLE
update general readme

### PR DESCRIPTION
**Pull request by Tabintel**
Added the README for Rhino UI, providing an overview of the project, its purpose, features, getting started, and information about the Rhino UI packages. The README is designed to help users understand and use Rhino UI effectively.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,81 +1,169 @@
...
-- `@repo/ui`: a stub React component library shared by both `web` and `docs` applications
-- `@repo/eslint-config`: `eslint` configurations (includes `eslint-config-next` and `eslint-config-prettier`)
-- `@repo/typescript-config`: `tsconfig.json`s used throughout the monorepo
+4. **Modular import pattern for optimized bundle size and performance**
+Rhino UI adopts a modular import pattern, allowing components to be imported directly from their subpaths or as an aggregate. This flexibility reduces bundle size, optimizes performance, and supports individual component exports. Examples include:
+   - Direct import: `import Alert from '@rhino-ui/components/alert'`
+   - Aggregate import: `import { Alert } from '@rhino-ui/components'`
 
-Each package/app is 100% [TypeScript](https://www.typescriptlang.org/).
+5. **Live testing of UI components with Docusaurus**
```
Comment (chrisllontop) [inline README.md:36]: I think this can be deleted for now.

File: README.md [hunk 2]
```diff
@@ -1,81 +1,169 @@
...
 
-This Turborepo includes the following packages/apps:
+2.  **Consistent code formatting and linting with ESLint and Prettier configurations**
+Setting up ESLint and Prettier for each project can be time-consuming. Rhino UI streamlines this process with its ESLint (`@rhino-ui/eslint-config`) and Prettier (`@rhino-ui/ts-config` for TypeScript projects) config modules. These configurations, located in the [libs](https://github.com/rhinolabs/rhino-ui/tree/main/libs) folder, are integrated into the [pnpm](https://github.com/rhinolabs/rhino-ui/blob/main/pnpm-workspace.yaml) workspace. This approach ensures consistency across projects, speeds up development, and maintains clean code.
 
-### Apps and Packages
+3. **Efficient component development, testing, and documentation with Storybook CS53 setup**
+Rhino UI uses Storybook CS53 setup, which is more efficient for developing, testing, and documenting UI components in isolation. This setup enhances collaboration and ensures the seamless integration of UI components into projects.
```
Comment (chrisllontop) [inline README.md:29]: We will remove Storybook, so we can delete this section for now.

**Review by chrisllontop — CHANGES_REQUESTED** [review]

**Comment by chrisllontop** [comment]
I think we could talk about the eslint, prettier, and stylelint but keep in mind this is not available for public usage, it is just to be used in RhinoUI development.

File: README.md (hunk 1 above)
Comment (Tabintel) [reply README.md]: Alright

**Comment by chrisllontop** [comment]
Hi @Tabintel, ts-config is not for public usage too. So, we can delete that section entirely.

Also, we can remove the usage section for components and design tokens. A simple explanation of the libraries is enough.

**Pull request merged**